### ISSUE TITLE
What genes are upregulated by chemical X goes the wrong way or in some way is messed up

### ISSUE THREAD

**Issue by dkoslicki** [RTXteam/RTX#2048]
Using Valproic acid and the [official UI](https://ui.test.transltr.io/results?l=Valproic%20Acid&t=3&q=a080dccc-66b2-4c08-9d66-443c593a8e9b), I see a result of `Abat` which is essentially: `Abat "Has Decreased Activity Or Abundance Caused By" Ube2c "Has Increased Expression Caused By" Valproic Acid`. However, looking at this same ID `a080dccc-66b2-4c08-9d66-443c593a8e9b` in [our UI](https://arax.ncats.io/beta/?r=141200), there 
<img width="1512" alt="Screenshot 2023-06-05 at 12 56 25 PM" src="https://github.com/RTXteam/RTX/assets/6362936/1d00bf89-b2bb-443c-8733-74c3975e92df">

You'll see that the support graph doesn't have this `Ube2c` path? So either: a) our UI isn't displaying support graphs correctly (@isbluis ?), or else b) something is getting lost in translation to the new TRAPI 1.4 paradigm (@chunyuma ?).

Lastly, note that this is a bad answer Valproic acid -[increases]-> Ube2c -[decreases]-> Abat, so Valproic acid downregulates chemical X, while it was asked for upregulates.

**Comment by edeutsch** [comment]
Revisit with David at 6/21 AHM

**Comment by edeutsch** [comment]
Touched on this briefly at 2025-04-23 AHM. @dkoslicki is this still an issue?

**Comment by ShannonRanio** [comment]
@dkoslicki is this still an issue?

**Comment by saramsey** [comment]
Can we close this, in light of PR #2772?

**Comment by dkoslicki** [comment]
Yes, this issue is now fixed with the new xCRG: https://arax.ci.transltr.io/?r=474457
